--- ./httpd.conf.org	2016-06-03 12:20:36.000000000 +0900
+++ httpd.conf	2016-06-03 12:23:34.000000000 +0900
@@ -49,7 +49,7 @@ ServerRoot "/usr/local/apache2"
 # prevent Apache from glomming onto all bound IP addresses.
 #
 #Listen 12.34.56.78:80
-Listen 80
+Listen {{ var "LISTEN_PORT" | default "80"  }}
 
 #
 # Dynamic Shared Object (DSO) Support
@@ -114,11 +114,11 @@ LoadModule headers_module modules/mod_he
 LoadModule setenvif_module modules/mod_setenvif.so
 LoadModule version_module modules/mod_version.so
 #LoadModule remoteip_module modules/mod_remoteip.so
-#LoadModule proxy_module modules/mod_proxy.so
+LoadModule proxy_module modules/mod_proxy.so
 #LoadModule proxy_connect_module modules/mod_proxy_connect.so
 #LoadModule proxy_ftp_module modules/mod_proxy_ftp.so
 #LoadModule proxy_http_module modules/mod_proxy_http.so
-#LoadModule proxy_fcgi_module modules/mod_proxy_fcgi.so
+LoadModule proxy_fcgi_module modules/mod_proxy_fcgi.so
 #LoadModule proxy_scgi_module modules/mod_proxy_scgi.so
 #LoadModule proxy_wstunnel_module modules/mod_proxy_wstunnel.so
 #LoadModule proxy_ajp_module modules/mod_proxy_ajp.so
@@ -190,7 +190,7 @@ ServerAdmin you@example.com
 #
 # If your host doesn't have a registered DNS name, enter its IP address here.
 #
-#ServerName www.example.com:80
+ServerName localhost:{{ var "LISTEN_PORT" | default "80" }}
 
 #
 # Deny access to the entirety of your server's filesystem. You must
@@ -214,8 +214,8 @@ ServerAdmin you@example.com
 # documents. By default, all requests are taken from this directory, but
 # symbolic links and aliases may be used to point to other locations.
 #
-DocumentRoot "/usr/local/apache2/htdocs"
-<Directory "/usr/local/apache2/htdocs">
+DocumentRoot "/var/www/html"
+<Directory "/var/www/html">
     #
     # Possible values for the Options directive are "None", "All",
     # or any combination of:
@@ -248,7 +248,7 @@ DocumentRoot "/usr/local/apache2/htdocs"
 # is requested.
 #
 <IfModule dir_module>
-    DirectoryIndex index.html
+    DirectoryIndex index.php index.html
 </IfModule>
 
 #
@@ -492,3 +492,7 @@ SSLRandomSeed startup builtin
 SSLRandomSeed connect builtin
 </IfModule>
 
+<IfModule proxy_fcgi_module>
+ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://{{ var "BACKEND" | default "php:9000" }}/var/www/html/$1
+</IfModule>
+
